docs/framework/configure-apps/file-schema/windows-identity-foundation/issuertokenresolver.md

```diff
@@ -64,9 +64,9 @@ Registers the issuer token resolver that is used by handlers in the token handle
 </issuerTokenResolver>  
 ```  
   
-## Example  
+## Example
   
-```  
+```csharp  
 public override void LoadCustomConfiguration(System.Xml.XmlNodeList nodelist)  
```

Comment:
The usage of `XmlNode` in the foreach loop indicates that there is already a `using System.Xml;` statement. Which means `System.Xml` is not needed here.

```suggestion
public override void LoadCustomConfiguration(XmlNodeList nodelist)
```